src/components/data-management/FileUploadModal.jsx

```diff
@@ -65,14 +70,14 @@ const FileUploadModal = (props) => {
 
   const [selectedTech, setSelectedTech] = useState(currentSelectedTech ?? sampleTech['10X']);
   const [canUpload, setCanUpload] = useState(false);
-  const [filesList, setFilesList] = useState([]);
+  const [files, setFiles] = useState(emptyFiles);
```

Comment:
They are always obtained and updated  together, so it seemed to make sense to put `invalid` in the same `useState`